### DISCUSSION TITLE
possibility to add/see Avatars in Virtual cockpit

### DISCUSSION THREAD

**Discussion by andrejetcat** [flybywiresim/aircraft#3073] [Q&A] [answered]
I don´t know if there is already a topic like this, but it is possible to add the pilots avatars in the cockpit? 
It is quite unrealistic to look to the right an see no copilot :D

**Comment by hotshotp** [comment, marked as answer]
This would be really bad in my opinion if that was to be implemented. The models are low quality as they are meant to be viewed from the outside. The models on the inside would have to be high quality and would take up lots of performance. Further on, the pilot would probably cover up some switches and buttons that are needed. It just isn't a good idea
